### PR TITLE
{perf}[gompi/2021.04] OSU-Micro-Benchmarks v5.7.1

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

depends on `gompi/2021.04` (~~#12571~~)

### CODE REVIEW

**Review by Micket — CHANGES_REQUESTED** [review]
There is a 5.7.1 version out, but, also, the websites SSL certificate is borked, so we need to add an exception and switch to http again I think (unless they fix it soon)

**Comment by branfosj** [comment]
> There is a 5.7.1 version out, but, also, the websites SSL certificate is borked, so we need to add an exception and switch to http again I think (unless they fix it soon)

I've switched to 5.7.1 and to using http.

File: easybuild/easyconfigs/o/OSU-Micro-Benchmarks/OSU-Micro-Benchmarks-5.7.1-gompi-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'OSU-Micro-Benchmarks'
+version = '5.7.1'
+
+homepage = 'http://mvapich.cse.ohio-state.edu/benchmarks/'
+description = """OSU Micro-Benchmarks"""
+
+toolchain = {'name': 'gompi', 'version': '2021a'}
```
Comment (boegel) [inline easybuild/easyconfigs/o/OSU-Micro-Benchmarks/OSU-Micro-Benchmarks-5.7.1-gompi-2021a.eb:9]: @branfosj Can you change this to `2021.04` (the temporary version for what is likely to become `gompi/2021a`)? Cfr. ~~#12571~~

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/o/OSU-Micro-Benchmarks/OSU-Micro-Benchmarks-5.7.1-gompi-2021a.eb (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/o/OSU-Micro-Benchmarks/OSU-Micro-Benchmarks-5.7.1-gompi-2021a.eb]: done in f5e9295

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/ce1603a9a30d4d46b1c68f44bab97e6e for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bber0501u03a.bb2.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/c68873d1934159509160e41d86552605 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/c2c2b4bc581323fb862d60637a4693b9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
select-pika-c6gd-2xlarge-0001 - Linux centos linux 8.3.2011, AArch64, ARM UNKNOWN (graviton2), Python 3.6.8
See https://gist.github.com/095dbeae5fc417dc40944e1a78955b89 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild2.novalocal - Linux centos linux 8.3.2011, POWER, IBM pSeries (emulated by qemu) (power9le), Python 3.6.8
See https://gist.github.com/01fff18eb9ef89efcc4f5df2ba70bcfc for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Pull request merged**